Features/Standard/ContentTypeIcon.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Content Type icons
+
+    @javascript
+    Scenario: I can see the generic Content Type icon attached to the Content item name
+        Given a Content Type "custom_content_type" exists
+        And a Content item of this type exists
+        When I display that Content item
```

Comment:
We should/must check what verb we have used for similar cases (might have been "view").